src/WattleScript.Interpreter/Tree/MemberUtilities.cs

```diff
@@ -37,14 +39,18 @@ public static void AddModifierFlag(ref MemberModifierFlags source, Token token)
             
             foreach (var combo in flagConflicts)
             {
-                if (source.HasFlag(combo.a) && source.HasFlag(combo.b))
+                if ((combo.appliesTo == WattleMemberType.Any || combo.appliesTo == memberType) && source.HasFlag(combo.a) && (combo.b == null || source.HasFlag(combo.b)))
```

Comment:
this can just be `combo.appliesTo.HasFlag(memberType)`

Reply:
Fixed in 1dc8315